Python/Product/Profiling/Profiling/ProfilingTargetView.cs

```diff
@@ -79,6 +84,7 @@ public ProfilingTargetView(IServiceProvider serviceProvider, ProfilingTarget tem
                 Project = new ProjectTargetView(template.ProjectTarget);
                 IsStandaloneSelected = false;
                 IsProjectSelected = true;
+                UseVTune = false;              // TODO -- why not allow projects?
```

Comment:
Good question! All we do with the project is use it to find all the settings we'd provide for the standalone target.